meerk40t/gui/icons.py

```diff
@@ -4174,11 +3012,11 @@ def make_geomstr(self, gc, path):
 )
 
-icons8_laser_beam_20 = icons8_laser_beam_52
+icons8_laser_beam = icons8_laser_beam
```

Comment:
Not technically wrong. But, not needed.

Reply:
Not sure from where that came from, will remedy it at the next iteration